crates/gosub_styling/tools/generate_definitions/webref/webref.go

```diff
@@ -444,3 +450,83 @@ func ProcessExtraValues(values []WebRefValue, pd *ParseData) {
+	"-webkit-border-bottom-left-radius":  "border-bottom-left-radius",
+	"-webkit-border-bottom-right-radius": "border-bottom-right-radius",
+	"-webkit-border-radius":              "border-radius",
+	"-webkit-border-top-left-radius":     "border-top-left-radius",
+	"-webkit-border-top-right-radius":    "border-top-right-radius",
+	"-webkit-box-align":                  "box-align",
+	"-webkit-box-flex":                   "box-flex",
+	"-webkit-box-ordinal-group":          "box-ordinal-group",
+	"-webkit-box-orient":                 "box-orient",
+	"-webkit-box-pack":                   "box-pack",
+	"-webkit-box-shadow":                 "box-shadow",
+	"-webkit-box-sizing":                 "box-sizing",
+	"-webkit-filter":                     "filter",
+	"-webkit-flex":                       "flex",
+	"-webkit-flex-basis":                 "flex-basis",
+	"-webkit-flex-direction":             "flex-direction",
+	"-webkit-flex-flow":                  "flex-flow",
+	"-webkit-flex-grow":                  "flex-grow",
+	"-webkit-flex-shrink":                "flex-shrink",
+	"-webkit-flex-wrap":                  "flex-wrap",
+	"-webkit-justify-content":            "justify-content",
+	"-webkit-mask":                       "mask",
+	"-webkit-mask-box-image":             "mask-border",
+	"-webkit-mask-box-image-outset":      "mask-border-outset",
+	"-webkit-mask-box-image-repeat":      "mask-border-repeat",
+	"-webkit-mask-box-image-slice":       "mask-border-slice",
+	"-webkit-mask-box-image-source":      "mask-border-source",
+	"-webkit-mask-box-image-width":       "mask-border-width",
+	"-webkit-mask-clip":                  "mask-clip",
+	"-webkit-mask-composite":             "mask-composite",
+	"-webkit-mask-image":                 "mask-image",
+	"-webkit-mask-origin":                "mask-origin",
+	"-webkit-mask-position":              "mask-position",
+	"-webkit-mask-repeat":                "mask-repeat",
+	"-webkit-mask-size":                  "mask-size",
+	"-webkit-order":                      "order",
+	"-webkit-perspective":                "perspective",
+	"-webkit-perspective-origin":         "perspective-origin",
+	"-webkit-text-size-adjust":           "text-size-adjust",
+	"-webkit-transform":                  "transform",
+	"-webkit-transform-origin":           "transform-origin",
+	"-webkit-transform-style":            "transform-style",
+	"-webkit-transition":                 "transition",
+	"-webkit-transition-delay":           "transition-delay",
+	"-webkit-transition-duration":        "transition-duration",
+	"-webkit-transition-property":        "transition-property",
+	"-webkit-transition-timing-function": "transition-timing-function",
+	"-webkit-user-select":                "user-select",
+	"font-stretch":                       "font-width",
+	"grid-column-gap":                    "column-gap",
+	"grid-gap":                           "gap",
+	"grid-row-gap":                       "row-gap",
+}
```

Comment:
Might make sense to put those into a json file.

Reply:
i don't want to tag along resource files if it's just a single one. We can change that later